### ISSUE TITLE
Section Headers in Match or Pit Metric Setup

### ISSUE THREAD

**Issue by melindamongar-commits** [frc2052/FRC-Krawler#90]
When creating metrics, can we create section headers to break questions from one another for ease of use? Example, visually break Auto questions from Teleop from Endgame?

**Comment by bherbst** [comment]
Unfortunately that is not currently possible, but it is a great idea! We will consider it for a future release.

**Comment by melindamongar-commits** [comment]
Yeah, I figure it would be a next year release sort of thing. If I figure out how to implement it, I will touch base with you. I thought it was a smallest task I could learn from. 

Of course I have other things I would like to see and wasn't sure if I should create tickets for them or not. I would love to edit an existing Metric to change the Label and it's configuration parameters like changing the list options or increasing the counter max entry. I would also love to be able to reorder the entries but there looks to be an existing ticket for that.

**Comment by bherbst** [comment]
Now available in FRC Krawler 4.0!
